### PR TITLE
JSDoc fixes

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
- Add missing function stubs in AvatarList and AvatarManager.
- Miscellaneous fixes noticed in passing.

### CODE REVIEW

**Comment by ctrlaltdavid** [comment]
### Test Plan

Proof read the JSDoc changes per the commits.

You can view these at:
- http://ctrlaltstudio.com/downloads/hifi-dev/jsdoc/AvatarList.html
- http://ctrlaltstudio.com/downloads/hifi-dev/jsdoc/AvatarManager.html
- file:///D:/Workspaces/CtrlAltStudio/HiFi-B/HiFi-B/tools/jsdoc/out/PickType.html

https://highfidelity.manuscript.com/f/cases/17223

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13672/HighFidelity-Beta-PR14145-2ba8dc3.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13672/HighFidelity-Beta-PR14145-2ba8dc3.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13672/HighFidelity-Beta-Interface-PR14145-2ba8dc3.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13672/HighFidelity-Beta-PR14145-2ba8dc3.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13672/HighFidelity-Beta-Interface-PR14145-2ba8dc3.exe

File: interface/src/avatar/AvatarManager.h [hunk 1]
```diff
@@ -151,6 +152,12 @@ class AvatarManager : public AvatarHashMap {
                                                                         const QVector<EntityItemID>& avatarsToInclude,
                                                                         const QVector<EntityItemID>& avatarsToDiscard);
 
+    /**jsdoc
+     * @function AvatarManager.findParabolaIntersectionVector
+     * @param {PickParabola} pick
+     * @param {Uuid[]} avatarsToInclude
+     * @param {Uuid[]} avatarsToDiscard
+     */
```
Comment (dback2) [inline interface/src/avatar/AvatarManager.h]: What about return value?

Comment (ctrlaltdavid) [reply interface/src/avatar/AvatarManager.h]: Fixed

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13967/HighFidelity-Beta-PR14145-3539381.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13967/HighFidelity-Beta-PR14145-3539381.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13967/HighFidelity-Beta-Interface-PR14145-3539381.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13967/HighFidelity-Beta-PR14145-3539381.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13967/HighFidelity-Beta-Interface-PR14145-3539381.dmg

**Pull request merged**